src/org/opendatakit/briefcase/ui/BriefcaseCLI.java

```diff
@@ -304,7 +304,7 @@ public void run() {
         importODK(storageDir, Paths.get(odkDir), Optional.empty());
 
       if (odkDir == null && server != null)
-        pullFormFromAggregate(storageDir, formid, username, password, server, false, false, false);
+        pullFormFromAggregate(storageDir, Optional.empty(), username, password, server, false, false, false);
```

Comment:
A couple of things should be changed here:
- Although this will sound a little bit awkward, since this is the legacy CLI, we should make sure the original behavior is maintained. 
  
  This means that, in this CLI, form_id is required i.e. it should fail if form_id is not added.

  I'd suggest adding this before calling `pullFormFromAggregate`:

  ```java
  if (formId == null)
    throw new BriefcaseException("You need to provide a form ID (legacy CLI)");
  ```
- In any case, `Optional.ofNullable(formId)` should be passed to the `pullFormFromAggregate` method.